consensus/istanbul/backend/handler.go

```diff
@@ -109,12 +120,12 @@ func (sb *backend) HandleMsg(addr common.Address, msg p2p.Msg) (bool, error) {
 				TD    *big.Int
 			}
 			if err := msg.Decode(&request); err != nil {
-				log.Debug("Proposer was unable to decode the NewBlockMsg", "error", err)
+				log.Error("BFT: unable to decode the NewBlockMsg", "error", err)
```

Comment:
line 110, `log.Debug` became `sb.logger.Debug`, why not here and at line 128?

Reply:
Updated to use `sb.logger.Debug`